src/clients/save_network.rb

```diff
@@ -214,7 +212,11 @@ def copy_from_instsys
       new_SCR = WFM.SCROpen("chroot=/:scr", false)
       WFM.SCRSetDefault(new_SCR)
 
-      ay_mode_configuration if Mode.autoinst
+      # this has to be done here (out of chroot) bcs:
+      # 1) udev agent doesn't not support SetRoot
+      # 2) original ifcfg file is copied otherwise to. It doesn't break things itself
```

Comment:
parse error in "original ifcfg file is copied otherwise to". "to where?". "too"?